src/components/Info-Tile.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import { Link, useStaticQuery, graphql } from 'gatsby';
+import Socials from './Socials';
+
+export default function InfoTile() {
+  const data = useStaticQuery(graphql`
+    query {
+      allMarkdownRemark(sort: {fields: [frontmatter___date], order: DESC}) {
+        edges {
+          node {
+            fields {
+              slug
+            }
+            frontmatter {
+              tags
+              title
+            }
+          }
+        }
+      }
+    }
+`);
+
+  const tags = {};
+  data.allMarkdownRemark.edges.forEach(({ node }) => {
+    node.frontmatter.tags.forEach((tag) => {
+      tags[tag] = tag;
+    });
+  });
+
+  const recentPosts = data.allMarkdownRemark.edges.map(({ node }, index) => {
```

Comment:
Instead of checking in a loop like this, it'd probably be nicer to just slice the array before mapping:

```js
const recentPosts = data.allMarkdownRemark.edges.slice(0, 10).map(({ node }) => (
  <Link
      to={node.fields.slug}
      key={node.fields.slug}
      className='col-12'
  >
      {node.frontmatter.title}
  </Link>
))